spiderable_server.js

```diff
@@ -4,6 +4,10 @@ var querystring = Npm.require('querystring');
 var urlParser = Npm.require('url');
 var crypto = Npm.require('crypto');
 
+var bound = Meteor.bindEnvironment(function(callback) {
```

Comment:
I definitely want to move this to coffee.
Personally, I prefer tabs over spaces too, but it doesn't matter as much.

I would wait until we are stable before converting. We wouldn't want to
introduce new code accidentally.

I'd prefer to do that restructuring, since I'd like to also add some
directory structure as well. lib/ and test/ for example.
It would be nice to clean up the original code.
On Jul 21, 2015 2:40 PM, "dr.dimitru" <notifications@github.com> wrote:

> In spiderable_server.js
> <https://github.com/jazeee/jazeee-meteor-spiderable/pull/7#discussion_r35144885>
> :
>
> > @@ -4,6 +4,10 @@ var querystring = Npm.require('querystring');
> >  var urlParser = Npm.require('url');
> >  var crypto = Npm.require('crypto');
> >
> > +var bound = Meteor.bindEnvironment(function(callback) {
>
> Okay, I'll work around this now.
> Next - send new pull request with caching.
> One more thing - what do you think about to move on CoffeeScript?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jazeee/jazeee-meteor-spiderable/pull/7/files#r35144885>
> .
>